Fabric.Identity.API/Configuration/IdentityConfigurationProvider.cs

```diff
@@ -77,6 +77,17 @@ private void DecryptEncryptedValues(IAppConfiguration appConfiguration, Decrypti
                     appConfiguration.AzureActiveDirectorySettings.ClientSecret,
                     appConfiguration.SigningCertificateSettings);
             }
+
+            if (appConfiguration.AzureActiveDirectoryClientSettings?.ClientAppSettings != null)
```

Comment:
So did we have this code before the .net core 2.2 upgrade but got missed with the changes?

Reply:
This, or the equivalent was in IdPSS, it was just missed when porting over